xml/docu_styleguide.structure.xml

```diff
@@ -1259,10 +1259,18 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
       <formalpara>
        <title>Prefix</title>
        <para>
-        Signifies the type of XML element. Use in accordance with
-        <xref linkend="tab-id-prefix"/>.
+        Signifies the type of XML element.
+        Prefixes are meant as an aid to writers to allow creating
+        logically-named, but distinct identifiers even for closely related
+        elements.
```

Comment:
Prefixes aid writers in creating logically named but distinct identifiers for elements.

Reply:
We could even simplify further:
```suggestion
        Prefixes aid writers in creating logically named identifiers for elements.
```